tests/phpunit/includes/doing-it-wrong-trait.php

```diff
@@ -34,7 +36,7 @@ public static function filter_doing_it_wrong_trigger_error() {
 	 * @return void
 	 */
 	public function doing_it_wrong_run( $function, $message = '', $version = '' ) {
-		if ( 'PLL_Model::get_languages_list()' === $function ) {
+		if ( 'WP_Syntex\Polylang\Models\Languages::get_languages_list()' === $function ) {
```

Comment:
Isn't it possible to use `use` directives in a trait?

Reply:
We can. You want to it `Languages::class . '::get_list()'`?

Reply:
No. It was a bad idea.